libraries/TeamSpeak3/Node/Server.php

```diff
@@ -89,6 +89,28 @@ public function request($cmd, $throw = TRUE)
 
     return $this->getParent()->request($cmd, $throw);
   }
+  
+  /**
+   * Returns an array filled with TeamSpeak3_Node_Server objects.
+   *
+   * @param  array $filter
+   * @return array
+   */
+  public function serverInfo()
+  {
+	return $this->request("serverinfo")->toList();
+  }
+  
+  /**
+   * Resets the server info.
+   *
+   * @return void
+   */
+  public function serverInfoReset()
+  {
+    $this->resetNodeList();
+    $this->serverInfo = null;
```

Comment:
`$this->serverInfo` is undefined in the context of `TeamSpeak3_Node_Server`.

Maybe you meant `$this->nodeInfo` defined in `TeamSpeak3_Node_Abstract`. In which case, you already have access to reset via `TeamSpeak3_Node_Server->resetNodeInfo()` as defined in: `TeamSpeak3_Node_Abstract->resetNodeInfo()`.
Ref: https://docs.planetteamspeak.com/ts3/php/framework/class_team_speak3___node___abstract.html#af0a532082869f2eb5c99951c4e9251b4

Also, note that `TeamSpeak3_Node_Server->nodeInfo` is initialized and reset to `(array) array()` not `null`.